gui/common/ssl.py

```diff
@@ -115,9 +112,9 @@ def create_certificate_signing_request(cert_info):
         # This is raised if say we specified freenas.org in the Common name
         pass
 
-    req.add_extensions([crypto.X509Extension(
-        "subjectAltName".encode('utf-8'), False, f"{default_san_type}:{cert_info['san']}".encode('utf-8')
-    )])
+    req.add_extensions([crypto.X509Extension(b"subjectAltName", False, f"{default_san_type}:{cert_info['san']}".encode())])
+    req.get_subject().subjectAltName = cert_info['san'].replace(" ", ", ")
+    req.get_subject().subjectAltName = cert_info['san']
```

Comment:
Two `subjectAltName` assignments? that looks strange

Reply:
I don't know how that happened....